airbyte-webapp/src/packages/cloud/views/workspaces/WorkspacesPage/components/WorkspaceItem.module.scss

```diff
@@ -0,0 +1,20 @@
+$white: #fff;
+$shadowColor: rgb(26 25 77 / 12%);
+
+.button {
+  padding: 20px 28px 20px 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 10px;
+  cursor: pointer;
+  width: 100%;
+  background: $white;
+  border-radius: 10px;
+  box-shadow: 0 2px 4px $shadowColor;
+  border: none;
+}
+
+.iconColor {
+  color: #615eff;
+}
```

Comment:
All these free-standing colors make me a touch nervous. Are we sure these colors can't come from `_colors.scss`?

Reply:
Oh I might have missed this one, sorry! I was having trouble importing `_colors.scss` in a few places, will go back and check.

Reply:
If you continue to have issues with it, feel free to shoot me a ping! `@use` is a little weird at first :)